README.md

```diff
@@ -51,4 +51,11 @@ yarn build
 5. To test the application
 ```
 yarn test
-```
\ No newline at end of file
+```
+
+### Note:
+
+ * By default speech assistant app will be enabled while building the app
+ * To disable the speech assistant app at build time, trigger the workflow with false value of Load SA App
```

Comment:
This README is not making much sense to me. 
We should first explain that what we mean when we say disable the app or what does it mean when workflow is run by false value.
It will be difficult for the person to understand, who doesn't have the context